aspnetcore/host-and-deploy/blazor/client-side.md

```diff
@@ -227,6 +227,20 @@ If a *500 - Internal Server Error* is received and IIS Manager throws errors whe
 
 For more information on troubleshooting deployments to IIS, see <xref:host-and-deploy/iis/troubleshoot>.
 
+### Azure Storage
+
+Static file hosting provided by Azure Storage permits serverless Blazor app hosting. Custom domain names, the Azure Content Delivery Network (CDN), and HTTPS are supported.
+
+When the blob service is enabled for static website hosting on a storage account:
+
+* Set the **Index document name** to `index.html`.
+* Set the **Error document path** to `index.html`. Razor Components and other non-file endpoints don't reside at physical paths in the static content stored by the blob service. When a request for one of these resources is received that the Blazor router should handle, the *404 - Not Found* error generated by the blob service routes the request to the **Error document path**. The *index.html* blob is returned, and the Blazor router loads and processes the path.
+
+For more information, see the following resources:
+
+* [Static website hosting in Azure Storage](/azure/storage/blobs/storage-blob-static-website)
+* [Serverless web apps with Blazor, Azure Functions, and Azure Storage (Jeff Hollan, Microsoft Build, 5/6/2019)](https://mybuild.techcommunity.microsoft.com/sessions/77336)
```

Comment:
As much as I love this presentation I prefer to not link to it from our official docs. Over time it will go stale.